torchrl/envs/transforms/transforms.py

```diff
@@ -209,6 +209,7 @@ class Transform(nn.Module):
     """
 
     invertible = False
+    disable_inv_on_reset = False
```

Comment:
Upon reflection: why not making this `True` by default?
Seems like it's more the exception than the rule (since we haven't been needing it so far)?

Reply:
Yeah that makes sense